packages/anthropic/src/anthropic-tools.ts

```diff
@@ -69,6 +69,54 @@ function bashTool_20241022<RESULT>(
   };
 }
 
+const Bash20250124Parameters = z.object({
+  command: z.string(),
+  restart: z.boolean().optional(),
+});
+
+/**
+ * Creates a tool for running a bash command. Must have name "bash".
+ *
+ * Image results are supported.
+ *
+ * @param execute - The function to execute the tool. Optional.
+ */
+function bashTool_20250124<RESULT>(
```

Comment:
Identical with `bashTool_20241022`, just the version change. The [tool imput schema](https://docs.anthropic.com/en/docs/agents-and-tools/computer-use#bash-tool) is the same for both